range-slider/src/app/app.component.ts

```diff
@@ -11,8 +12,20 @@ export class AppComponent {
   public someMin: number = -10;
   public someMax: number = 10;
 
+  @ViewChild('slider') slider: NouisliderComponent;
+
   onChange(value: any) {
     console.log('Value changed to', value);
+    this.someValue = [value[0], this.someValue[1]];
+  }
```

Comment:
What about 
`this.someValue[0]=value[0]` ?

Is it necessary to assign both entries to someValue, even if only one changed?